tests/format/vue/v-for/ts.vue

```diff
@@ -0,0 +1,10 @@
+<script lang="ts"></script>
+<template>
+<div
+v-for="a:number       of x as number[]"
+v-for="[ a  , b]   : [   string,string ]     of x as Array<     [string, string]>"
+v-for="  a  of list.map( (x:any): unknown     => x.foo.bar)"
+v-for="([longLongProp, longLongProp, [longLongProp, longLongProp='Hello, Prettier!', [longLongProp, longLongProp, anotherLongLongProp=[longLongProp, longLongProp, anotherLongLongProp, yetAnotherLongLongProp], yetAnotherLongLongProp], yetAnotherLongLongProp], yetAnotherLongLongProp], index) of longLongLongLongLongLongLongLongList"
+>
+</div>
+</template>
```

Comment:
nit: add a new line at last.

Reply:
Thanks, I found that I had disabled editorconfig plugin mistakenly 😅